overlord/snapstate/snapstate.go

```diff
@@ -1698,6 +1699,7 @@ func ActiveInfos(st *state.State) ([]*snap.Info, error) {
 		snapInfo, err := snapst.CurrentInfo()
 		if err != nil {
 			logger.Noticef("cannot retrieve info for snap %q: %s", snapName, err)
+			fmt.Fprintf(os.Stderr, "[stderr] cannot retrieve info for snap %q: %s\n", snapName, err)
```

Comment:
this calls readInfoAnyway so don't need this logging here, we get things twice now otherwise